test/functorch/test_vmap.py

```diff
@@ -317,6 +317,49 @@ def foo(x):
         result = vmap(foo, out_dims=(1,))(tensor)
         self.assertEqual(result, expected)
 
+    def test_out_dims_none_tuple(self):
+        def foo(x):
+            return x, 'hello world'
+
+        tensor = torch.randn(2, 3)
+        result = vmap(foo, out_dims=(0, None))(tensor)
+        self.assertEqual(result[1], 'hello world')
+        self.assertIsInstance(result[0], torch.Tensor)
```

Comment:
Would using `self.assertEqual(result[0], tensor)` make more sense as it is more stricter that `assertIsInstance`?

Reply:
added